cratedb_toolkit/testing/testcontainers/cratedb.py

```diff
@@ -48,21 +50,21 @@ class CrateDBContainer(DbContainer):
     CRATEDB_USER = os.environ.get("CRATEDB_USER", "crate")
     CRATEDB_PASSWORD = os.environ.get("CRATEDB_PASSWORD", "")
     CRATEDB_DB = os.environ.get("CRATEDB_DB", "doc")
-    CRATEDB_KEEPALIVE = os.environ.get("CRATEDB_KEEPALIVE", os.environ.get("TC_KEEPALIVE", False))
+    KEEPALIVE = asbool(os.environ.get("CRATEDB_KEEPALIVE", os.environ.get("TC_KEEPALIVE", False)))
 
     # TODO: Dual-port use with 4200+5432.
     def __init__(
         self,
+        # TODO: Use `crate/crate:nightly` by default?
         image: str = "crate:latest",
```

Comment:
I think we should do it, to always use the most recent code for testing all downstream applications, in order to be able to discover regressions early.

Reply:
Implemented with 1c49f92f8d.